org.eclipse.winery.repository.ui/package.json

```diff
@@ -28,6 +28,7 @@
         "@angular/platform-browser-dynamic": "4.3.5",
         "@angular/router": "4.3.5",
         "core-js": "2.5.0",
+        "file-loader": "^0.11.2",
```

Comment:
Why do you need this library? If you really need it, please do not add `^` in front. Maybe you have seen it at the other lines, that we require the exact version. This is due to the Eclipse IP process, which we discusses at our initial meetings.